frontend/src/component/events/EventTimeline/EventTimelineEventGroup/EventTimelineEventGroup.tsx

```diff
@@ -31,9 +32,13 @@ export const EventTimelineEventGroup = ({
     const eventTime = group[0].timestamp;
 
     const position = `${((eventTime - startTime) / timelineDuration) * 100}%`;
+    const { trackEvent } = usePlausibleTracker();
```

Comment:
Should we move this to the start of this component's code? That's what we usually do with hooks. So, in this case it would be before the `const timelineDuration = endTime - startTime;`